checkrebuild

```diff
@@ -22,17 +22,18 @@ get_unofficial_pkgs() {
     comm -23 <(pacman -Qq | sort) <(pacman -Sql "${repos_to_skip[@]}" 2>/dev/null | sort)
 }
 
-filter_executable() {
-    while read -r file; do
-        [[ -f "$file" && -x "$file" ]] && echo "$file"
-    done
-}
-
 get_package_files() {
     xargs pacman -Qql |
     perl -pe 's/\n/\0/' |
-    xargs -0 readlink -f |
-    filter_executable
+    xargs -0 readlink -ez |
+    sort -uz
+}
+
+filter_executable() {
+    LANG=C xargs -0 stat --printf "%F %a\t%n\0" |
+    grep -ozP "^regular file ([1-6]|)[1357].*\t\K.*" |
+    xargs -0 file |
+    grep -oP ".*(?=: *ELF )"
```

Comment:
The `*` before `ELF`, did you see an example where there was no space, or it's just a precaution? I'd remove it unless we know it is needed, the simpler regex is, the better 😄

Reply:
Will work, but first have to disable default padding behavior in `file` call.

Reply:
Aaaah, I see now why it was there - but yeah, even cooler without padding!